pkg/volume/azure_file/azure_file.go

```diff
@@ -19,6 +19,7 @@ package azure_file
 import (
```

Comment:
azure_file change should not be included?

Reply:
It's based on azurefile PR, when it's merged, there should be no this code change

Reply:
actually I have changed Mount func in mount_windows.go, which will affect azure_file code